From 3019628843fecf84f01353f26b618cf74e23b93b Mon Sep 17 00:00:00 2001
From: Thomas Lamprecht <t.lamprecht@proxmox.com>
Date: Wed, 6 Apr 2016 16:46:11 +0200
Subject: [PATCH 51/52] net: NET_CLIENT_OPTIONS_KIND_MAX changed

s/NET_CLIENT_OPTIONS_KIND_MAX/NET_CLIENT_OPTIONS_KIND__MAX/
---
 net/net.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/net.c b/net/net.c
index 8580cc0..369d587 100644
--- a/net/net.c
+++ b/net/net.c
@@ -1333,7 +1333,7 @@ int64_t qmp_get_link_status(const char *name, Error **errp)
     bool ret;
 
     queues = qemu_find_net_clients_except(name, ncs,
-                                          NET_CLIENT_OPTIONS_KIND_MAX,
+                                          NET_CLIENT_OPTIONS_KIND__MAX,
                                           MAX_QUEUE_NUM);
 
     if (queues == 0) {
-- 
2.1.4

